Shared/Tests.Shared/RealmInstanceTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System.IO;
+using Realms;
+
+namespace IntegrationTests
+{
+    [Preserve(AllMembers = true)]
+    public abstract class RealmInstanceTest : RealmTest
+    {
+        protected RealmConfiguration _configuration = new RealmConfiguration(Path.GetTempFileName());
```

Comment:
Why is this different from `RealmConfiguration.DefaultConfiguration` and in which cases should I use `Realm.GetInstance()` instead of `Realm.GetInstance(_configuration)`?